oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -150,6 +150,51 @@ ADRs
 .. _invaluable technique: https://www.thoughtworks.com/radar/techniques/lightweight-architecture-decision-records
 .. _edx/credentials/docs/decisions: https://github.com/edx/credentials/tree/master/docs/decisions
 
+A suggested ADR template::
```

Comment:
This ones optional but making this a code block might look a little better.
```suggestion
A suggested ADR template:

.. code-block:: rst
```

Reply:
The double colon makes it a code block! `::` But I like your way; it’s more explicit.